src/Interfaces/PopulatorInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Search\Interfaces;
+
+interface PopulatorInterface
+{
+    /**
+     * Populates a handler's index with an array of objects.
+     *
+     * @param \LoyaltyCorp\Search\Interfaces\TransformableSearchHandlerInterface $handler
+     * @param string $indexSuffix
+     * @param int $batchSize
+     *
+     * @return void
+     */
+    public function populate(
+        TransformableSearchHandlerInterface $handler,
+        string $indexSuffix,
+        int $batchSize
+    ): void;
+
+    /**
+     * Populates a handler's index with an array of objects.
```

Comment:
The method description is identical to `populate()`, could it be different for `populateWith()`?

Reply:
Good catch